src/users/interface/userInterface.ts

```diff
@@ -5,5 +5,10 @@ export interface IUser {
   firstName: string;
   lastName: string;
   email: string;
+  password: string;
+  isAdmin: boolean;
+  salary: number;
+  avatar: string;
+  avatar_public_id: string;
```

Comment:
One part is kebabCase one part is underscore case, please correct it as per logic